Tasks/Samkevich/Task1/Task1.py

```diff
@@ -0,0 +1,13 @@
+deposit_amount = float(input('Enter deposit amount:\n'))
+deposit_term = float(input('Enter deposit term in years:\n'))
+annual_percent = float(input('Enter annual percent:\n'))
```

Comment:
the decimal would be more suitable here, since the task is money-related, -1 point for the unwise choice